src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -970,16 +973,26 @@ pub struct GraphEditorModel {
 // === Public ===
 
 impl GraphEditorModel {
-    pub fn new<S:Into<Scene>>(scene:S, cursor:cursor::Cursor, network:&frp::Network) -> Self {
-        let scene          = scene.into();
+    pub fn new
+    ( world   : &World
+    , cursor  : cursor::Cursor
+    , network : &frp::Network
+    ) -> Self {
+        let scene          = world.scene().clone_ref();
         let logger         = Logger::new("GraphEditor");
         let display_object = display::object::Instance::new(&logger);
         let nodes          = Nodes::new(&logger);
 //        let visualizations = Stage::new(scene.clone_ref(), Logger::new("VisualisationCollection"));
         let edges          = default();
         let frp            = FrpInputs::new(network);
         let touch_state    = TouchState::new(network,&scene.mouse.frp);
-        Self {logger,display_object,scene,cursor,nodes,edges,touch_state,frp}//visualizations }
+        let project_name   = component::ProjectName::new(world);
+        scene.add_child(&display_object);
+        scene.add_child(&project_name);
```

Comment:
this is also bad - a component, like graph editor model, should not add things to scene, which is parent of the components. You can add project name as child to graph editor, but not this way

Reply:
I had one main reason to do it: It was a workaround to display the `ProjectName` in the same position both for debug shapes and IDE.

As an action for your comment, I added it to `display_object` instead, but I fixed the position of `GraphEditor` on `IDE`. Which also fixes the connection lines not following the cursor.

I will need to ask @farmaazon and @mwu-tow to review this PR again.

![connections](https://user-images.githubusercontent.com/1069140/87093752-16732200-c214-11ea-8d66-c9732cdd4a87.gif)